setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml

```diff
@@ -48,14 +48,15 @@
   sudo: Yes
   apt: update_cache=yes
 
-- name: Install python Enthought dependencies
+- name: Install python dependencies
   sudo: True
   command: >
              pip install traits traitsui pyserial pylibftdi
-             pyparsing==1.5.7 pygments intelhex six construct sbp==0.27
```

Comment:
... so it does bother me that the `--allow-unverified` is needed here, I don't think that intelhex will install otherwise. Is that no longer the case?